assets/prometheus-k8s/rules.yaml

```diff
@@ -99,17 +99,22 @@ spec:
   - name: kube-apiserver.rules
     rules:
     - expr: |
-        histogram_quantile(0.99, sum(rate(apiserver_request_latencies_bucket{job="apiserver"}[5m])) without(instance, pod)) / 1e+06
+        sum(rate(apiserver_request_latencies_sum{subresource!="log",verb!~"LIST|WATCH|WATCHLIST|PROXY|CONNECT"}[5m])) without(instance, pod)
```

Comment:
we have all these metrics in 3.11 already?

Reply:
yes, we do. I was testing this on CI cluster which at that time was 3.11.